### DISCUSSION TITLE
Latex `\cong` is opposite to Chinese national standard

### DISCUSSION THREAD

**Discussion by lyxabymxlc** [KaTeX/KaTeX#3046] [Q&A]
The style rendered by latex `\cong` is opposite to that of Chinese national standard. 
How can I render the style of Chinese national standard ?
The style is as follows:
**Katex:**  
![image](https://user-images.githubusercontent.com/16789592/119790928-ee151480-bf06-11eb-8aba-a32f30dc945a.png)

**Chinese national standard:** 
![image](https://user-images.githubusercontent.com/16789592/119790878-de95cb80-bf06-11eb-9558-03f8ffa4734f.png)

**Comment by edemaine** [comment]
How do you get that symbol in LaTeX? Is there a package that defines it?

The closest one I see in [the symbol list](http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf) is `\backgong`, though it's not exactly the same.  Unfortunately, that symbol isn't supported by KaTeX (yet).
